lib/restclient/payload.rb

```diff
@@ -116,9 +116,12 @@ def inspect
       end
 
       def short_inspect
-        (size > 500 ? "#{size} byte(s) length" : inspect)
+        if RestClient.limit_payload_log? && size > 500
```

Comment:
I moved the 500 into a constant.
Currently, I don't see a real value for a setter.

The size refers to the bytesize of the string.